src/ncp/ncp_base.cpp

```diff
@@ -645,11 +647,10 @@ void NcpBase::UpdateChangedProps(void)
         }
         else if ((mChangedFlags & (OT_THREAD_CHILD_ADDED | OT_THREAD_CHILD_REMOVED)) != 0)
         {
-            // TODO: Uncomment this once we add support for this property.
-            //SuccessOrExit(HandleCommandPropertyGet(
-            //    SPINEL_HEADER_FLAG | SPINEL_HEADER_IID_0,
-            //    SPINEL_PROP_THREAD_CHILD_TABLE
-            //));
+            SuccessOrExit(HandleCommandPropertyGet(
+                SPINEL_HEADER_FLAG | SPINEL_HEADER_IID_0,
+                SPINEL_PROP_THREAD_CHILD_TABLE
+            ));
```

Comment:
Thinking more about this, I don't think updating the child table on the host whenever it changes is a good idea. This will cause the host to wake up every time a child associates or disassociates. Since there is nothing actionable for the host to do with this information, I think we should remove these four lines.